packages/dso-toolkit/reference/render/zoeken-in-lijst-cards.html

```diff
@@ -71,7 +71,9 @@ <h2 class="dso-steps-indicator">
         </div>
         <div class="row">
           <div class="col-md-12">
-            <h5>Actieve filters: <span class="dso-badge badge-success">Omgevingswet document</span><span class="dso-badge badge-warning">Geldige oude wetgeving</span></h5>
+            <h5>Actieve filters: <dso-badge level="success">Omgevingswet document</dso-badge>
+              <dso-badge level="warning">Geldige oude wetgeving</dso-badge>
```

Comment:
Bijzonder dat de codeformatter van `gulp reference:dom` deze twee de `<dso-badge>` op een eigen regel zet.. Laat maar even voor nu, is een aandachtspunt